CHANGELOG/CHANGELOG-0.7.md

```diff
@@ -0,0 +1,20 @@
+# Changelog 0.7
+
+## 0.7.0 - 2023-07-26
+
+### Added
+
+- Added new EgressName, EgressIP fields in flow table. ([#349](https://github.com/antrea-io/theia/pull/349), [@dreamtalen])
+
+### Fixed
+
+- Created a readonly user for Clickhouse operations ([#336](https://github.com/antrea-io/theia/pull/336), [@yanjunz97])
+
+### Upgraded
```

Comment:
Don't we use `Changed` as the section name usually (and position it between `Added` and `Fixed`)?

Reply:
I was just referring to the fact that we usually include dependency upgrades in the `Changed` section (in Antrea). I don't see any `Upgraded` section in other Theia Changelogs, so is this a new convention for Theia?

Reply:
ok, let me switch the upgraded section to changed section